privateSBOMExchange/src/petra/lib/internals/kms.py

```diff
@@ -22,21 +20,70 @@ class KeyManagementService:
     - Authenticates users via OIDC tokens (ambient/interactive).
     - Retrieves user attributes based on identity namespace.
     - Requests signing certificates from Fulcio. """
-    def __init__(self, kms_conf, key_lifetime_hours=24*356):
+    def __init__(self, kms_conf, key_lifetime_hours=24*30):
         self.kms_conf = kms_conf
         self.KEY_LIFETIME_HOURS = key_lifetime_hours
-        self.current_expiry_time = int(time.time()) + self.KEY_LIFETIME_HOURS * 3600
+        self.epoch_period_sec = key_lifetime_hours * 3600
```

Comment:
Converting back and forth between hours and seconds feels brittle. We should choose the same unit everywhere to minimize conversion bugs.